bower.json

```diff
@@ -20,7 +20,6 @@
   ],
   "dependencies": {
     "requirejs": "~2.1.15",
-    "requirejs-text": "~2.0.12",
-    "mobify-velocity": "~1.1.2"
```

Comment:
@marlowpayne I'm not sure we can remove that. I see a console error telling me `app/velocity.js` is missing. Let's discuss.